convert2rhel/actions/conversion/preserve_only_rhel_kernel.py

```diff
@@ -79,18 +79,18 @@ def run(self):
         """
         super(FixInvalidGrub2Entries, self).run()
 
+        loggerinst.task("Fix GRUB2 bootloader entries")
         if system_info.version.major < 8:
             # Applicable only on systems derived from RHEL 8 and later, and systems using GRUB2 (s390x uses zipl)
+            loggerinst.info("Applicable only to RHEL 8 and newer.")
```

Comment:
Did we say anything about skipping or words like that? I forget
```suggestion
            loggerinst.info("Skipped. Only relevant to RHEL 8 and newer.")
```